packages/lib-classifier/src/store/TranscriptionReductions/TranscriptionReductions.js

```diff
@@ -0,0 +1,125 @@
+import asyncStates from '@zooniverse/async-states'
+import { GraphQLClient } from 'graphql-request'
+import { flow, types } from 'mobx-state-tree'
+
+export const caesarClient = new GraphQLClient('https://caesar-staging.zooniverse.org/graphql')
```

Comment:
Likewise, what’s the best way to set up staging/production clients here? Pass them into the store environment, like the Panoptes client?

Reply:
Could you please elaborate on how to do that? The classifier doesn’t seem to have a concept of staging vs. production environment.

Reply:
I'm thinking move the caesar client into `lib-panoptes-js`, where the environment is defined?

Reply:
I'd prefer to keep that library scoped to API requests to Panoptes. We could put together a caesar client or make the classifier code aware of the environment as needed just for this. Things like clients get passed into the store environment parameter as it essentially functions like a global.

Reply:
I found the environment this afternoon: in the root Classifier component? Working on that now but it'll take a while as all the tests have broken.